src/pages/event/event.tsx

```diff
@@ -3,13 +3,12 @@ import React, { FunctionComponent, useState } from "react";
 import { isFuture, isPast } from "date-fns";
 import { Helmet } from "react-helmet";
 import { ResourceEvent, EventProps } from "../../components/UI/ResourceEvent";
-import { importAll } from "../../common/utils/importAll";
 import { Pagination, getPaginatedPosts, getPaginatedPagesTotal } from "@govtechsg/tradetrust-ui-components";
-import { getSortedByDateDesc } from "../../utils";
+import { getSortedByDateDesc, getCmsContentWithSlug } from "../../utils";
 import { Page } from "../../components/Layout/Page";
 
-let events = importAll(require.context("../../../cms/event/", false, /\.md$/)) as EventProps[];
-events = getSortedByDateDesc(events);
+const rawEvents = getCmsContentWithSlug(require.context("../../../cms/event/", false, /\.md$/));
+export const events = getSortedByDateDesc(rawEvents);
```

Comment:
try to put this outside in an index file, and import it in here rather than declaring it here to be shared with event and eventdetails.

Reply:
ok